src/api/app/views/webui2/webui/project/_pulse_list_comments.html.haml

```diff
@@ -0,0 +1,10 @@
+- if @comments.any?
+  - people = @comments.group(:user_name).count
+  %b
+    = pluralize(people.count, 'person', plural: 'people')
```

Comment:
The `plural` is not needed here, it works already without it.

![example](https://user-images.githubusercontent.com/1102934/49529438-4d55b900-f8b6-11e8-977a-cb72ed3d220a.png)

Reply:
Still valid